packages/utilities/psammead-locales/CHANGELOG.md

```diff
@@ -3,7 +3,6 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
-| 2.1.1 | [PR#1992](https://github.com/bbc/psammead/pull/1992) update the package with the side-effects flag |
```

Comment:
Sadly because of how psammead publishes, we need to treat this a fix forward, not a revert, so we need a new version number (2.1.2)